### DISCUSSION TITLE
What is day-to-day life of maintainer like?

### DISCUSSION THREAD

**Discussion by kshitij12345** [github/india#36] [Open Source] [answered]
More of coding? More of discussions and reviews?

What are good days like? What are bad days like?

**EDIT**: 
* How do you manage feature creep?

@sidharthachatterjee
@liyasthomas
@shikhamis11

**Comment by liyasthomas** [comment, marked as answer]
Starting the day with a positive thought is always helpful. Apart from coding, discussions and reviews - finding time to interact / speak / engage with the actual community members is what excites me the most. So for me - good days are those where I was able to interact with community more than actually writing code.

**Reply by adityaruplaha** [reply]
Personally, I alternate between review days and coding days. But I'm just starting off so I don't really have a solid idea.

**Reply by kshitij12345** [reply]
How do you manage feature creep? I would assume not every requested feature will make in. Is it community poll to decide which feature makes in or maintainers decide that?

**Reply by liyasthomas** [reply]
So whenever a new feature request arise -we immediately hop on a discussion and get to know about it's use cases. We'll review the request internally and create a poll in our community forums. After evaluating the feature request we'll usually implement a MVP and share a deploy preview with the community. This allow us to iterate over it and make sure it's a essential feature. We're always keen to keeping the project bloat-free and essentially minimal.

**Reply by kshitij12345** [reply]
That makes sense. Thanks!
